packages/thirdweb/src/pay/utils/commonTypes.ts

```diff
@@ -16,3 +16,5 @@ export type PayOnChainTransactionDetails = {
   completedAt?: string; // ISO DATE
   explorerLink?: string;
 };
+
+export type FiatProvider = "STRIPE" | "TRANSAK" | "KADO";
```

Comment:
what happens on the backend if i force stripe, but im not in the us?

Reply:
Hmm, good point. It will return a "Route not found" error